src/plugins/js-eval/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:slim
-RUN npm i --prefix /run date-fns airbnb-js-shims full-icu
+RUN npm i --prefix /run airbnb-js-shims full-icu acorn nearley
+# acorn nearley are needed by engine262
+RUN curl https://engine262.js.org/engine262.js > /run/node_modules/engine262.js
```

Comment:
I don't really like the build/deployment relying on engine262.js.org, and always grabbing the latest file hosted there. We need some kind of other solution, like installing from npm or vendoring the file and a script to update it.

Reply:
Ok good points, I added test directly in Dockerfile